Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -14,217 +14,297 @@
 //
 //******************************************************************************
 
-#include "Starboard.h"
+#import <Foundation/NSException.h>
+#import <NSOrderedSetInternal.h>
+#import <Starboard.h>
 #import <StubReturn.h>
+#import <VAListHelper.h>
 
-#include "Foundation/NSOrderedSet.h"
+// Needed to make sure that the returned array behaves like a one way proxy to an array
+@implementation _NSProxyOrderedSetArray
+- (instancetype)initWithOrderedSet:(NSOrderedSet*)orderedSet {
+    if (self = [super init]) {
+        _array = orderedSet->_arrayContainer;
```

Comment:
please prefer the of StrongId

---
In reply to: [63978985](https://github.com/Microsoft/WinObjC/pull/482#discussion_r63978985) [](ancestors = 63978985)